packages/Input/src/Input.js

```diff
@@ -64,56 +64,45 @@ const Input = React.forwardRef((props, ref) => {
     const iconSize = size === types.LARGE ? types.MEDIUM : types.SMALL;
 
     return (
-      <Button.Icon
+      <sc.ClearButton
```

Comment:
Since we don't provide class names anymore, maybe we should add some `pka-anchors` for our QA friends since they probably have been using the class name for automation tests

Reply:
This is a good idea.  I did add some, but the clear button would certainly benefit from one.